packages/bsky/tests/views/posts-debug.test.ts

```diff
@@ -0,0 +1,56 @@
+import { AtpAgent } from '@atproto/api'
+import { SeedClient, TestNetwork, basicSeed } from '@atproto/dev-env'
+import { ids } from '../../src/lexicon/lexicons'
+
+jest.mock('../../dist/hydration/util.js', () => {
+  const originalModule = jest.requireActual('../../src/hydration/util.ts')
+  return {
+    ...originalModule,
+    isDebugFieldAllowed: jest.fn(() => true),
+  }
+})
```

Comment:
If we can use the config and avoid jest module mocks that would rule.  I'd probably even prefer doing something a little dirty like reaching into `network.bsky.ctx.cfg` to add alice's DID.